apps/jsii-docgen/test/__fixtures__/libraries/construct-library/turbo.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "compile": {
+      "inputs": ["lib/**"],
+      "outputs": [".jsii"]
+    },
+    "package": {
+      "outputs": ["dist/**"]
+    }
```

Comment:
I'm not sure we need to package this library at all

Reply:
If we don't need to package it, I'll just remove the package script from package.json and its turbo config.